litepaper.md

```diff
@@ -21,6 +21,13 @@ The third group is a set of applications which can't work if the smart contracts
 ## Example Use Cases
 Obscuro supports transactional and computational privacy, which opens up a whole new world of use-cases.
 
+### Programmable privacy for DeFi
+Liquidation Hunting is where whales hunt for loans close to liquidation and push prices down to force liquidation. This behaviour creates huge volatility for everyone and a miserable experience for users forced into liquidation. Obscuro's programmable privacy allows developers to choose what they want to keep private and what they want to make public at a granular level. How does this apply to Liquidation Hunting? A developer can take an existing borrowing protocol and choose to have everything public, except for individual liquidation levels. Only if the liquidation levels get hit do they become public—an example of a simple change resulting in a much fairer protocol.
+
```

Comment:
need a space between "public" and the hyphen / dash otherwise "public-an" looks like a hyphenated word